Swiftfin/Components/EpisodesRowView/EpisodesRowView.swift

```diff
@@ -29,7 +29,8 @@ struct EpisodesRowView<RowManager>: View where RowManager: EpisodesRowManager {
 					}
 				} else {
 					Menu {
-						ForEach(Array(viewModel.seasonsEpisodes.keys).sorted(by: { $0.name ?? "" < $1.name ?? "" }), id: \.self) { season in
+						ForEach(Array(viewModel.seasonsEpisodes.keys).sorted(by: { $0.indexNumber ?? 0 < $1.indexNumber ?? 0 }),
```

Comment:
It may be better to implement a `Comparable` protocol or to be in charge of logic in ViewModel.

Reply:
You're right, good catch. I'll do that now instead